hooks/utils.py

```diff
@@ -0,0 +1,642 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2017, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
```

Comment:
Will this mean changing the license of this project?  (I have no problem with the GPL, just pointing it out).

Patching this file would avoid this.

Reply:
Thanks for pointing this out (I'll plead that I was doing this at an odd hour and failed to think things through).  I'll change this to patching (and also fix the comment typo).